packages/block-library/src/image/style.scss

```diff
@@ -88,7 +88,7 @@
 	// Supply caption styles to images, even if the theme hasn't opted in.
 	// Reason being: the new markup, <figcaptions>, are not likely to be styled in the majority of existing themes,
 	// so we supply the styles so as to not appear broken or unstyled in those themes.
-	figcaption {
+	:where(figcaption) {
```

Comment:
Is it intentional to lower it to 0,0,0 specificity, or should it also be prefixed with `:root` to make it `0,1,0`?

Reply:
It's not `0-0-0`, it's wrapped with `.wp-block-image`, so it's `0-1-0`.

Reply:
Oh yep, I should've expanded the whole file 😄